### PR TITLE
Refactor to improve code quality

### LINKED ISSUE #78: Refactor Storage class and code
- Improve code quality by refactoring code related to storage
- Rename storage to FileHandler to match its purpose

### PR DESCRIPTION
**Pull request by ehz0ah**
- Changed storage class to FileHandler
- Refactor FindOurSEP.java to ensure SLAP

Close #78

### CODE REVIEW

**Comment by ehz0ah** [comment]
WIP, DO NOT MERGE YET! Still testing the different file formats.

**Comment by ehz0ah** [comment]
Ready for review 👌

File: src/main/java/filehandler/FileHandler.java [hunk 1]
```diff
@@ -63,29 +62,34 @@ private String getFileExtension(File file) {
...
+            break;
+        case "json":
+            result = hasProcessJSON(path.toFile());
+            break;
+        case "txt":
+            result = hasProcessTXT(path);
+            break;
+        default:
+            throw SEPUnknownException.rejectUnknownFileType();
+        }
```
Comment (isaacsaw25) [inline src/main/java/filehandler/FileHandler.java:91]: Good use of switch case 👍

File: text-ui-test/EXPECTED.TXT [hunk 2]
```diff
@@ -1,12 +1,12 @@
 --------------------------------------------------------------------------------
-Hi! Welcome to FindOurSEP!
+Hi! Welcome to FindOurSEP! Enter 'help' for the list of commands.
```
Comment (isaacsaw25) [inline text-ui-test/EXPECTED.TXT:2]: Great to give the user explicit instructions 😄

**Review by isaacsaw25 — APPROVED** [review]
Excellent refactoring to make the code clearer and user experience better 😺

**Pull request merged**